.github/workflows/build-test-e2e.yml

```diff
@@ -85,6 +99,12 @@ jobs:
         run: yarn install --frozen-lockfile
       - name: Install Playwright Browsers
         run: yarn playwright install --with-deps
+      - name: Confirm Docker container is running
```

Comment:
Trying to troubleshoot the `TypeError: The argument 'file' cannot be empty. Received ''` error, as seen in [this run](https://github.com/hallieswan/SynapseWebClient/actions/runs/7425928398/job/20208782799). I can reproduce the error when the SWC war does not exist or when the docker container is not running (likely because in CI, we pass a stub to the [PW webServer command](https://github.com/Sage-Bionetworks/SynapseWebClient/blob/6f689abdaaa696d8664a380c1e2089fe43304d2f/playwright.config.ts#L93)).